Source/WebCore/Modules/mediastream/MediaStream.cpp

```diff
@@ -374,6 +378,11 @@ Document* MediaStream::document() const
     return downcast<Document>(scriptExecutionContext());
 }
 
+RefPtr<Document> MediaStream::protectedDocument() const
```

Comment:
I doubt we need this. We rarely need `protectedFoo()` functions for things that return a RefPtr (and not a Ref). This is because they usually need a null check and thus a local variable.